Keas.Mvc/Controllers/AccessController.cs

```diff
@@ -88,8 +93,10 @@ public string GetTeam()
             {
                 _context.Access.Add(access);
                 await _context.SaveChangesAsync();
+                await _eventService.TrackCreateAccess(access, await _securityService.GetUser());
+                return Json(access);
             }
-            return Json(access);
```

Comment:
@laholstege Can you confirm this change is correct?

Reply:
@jcubbage looks good